backend/cds/views.py

```diff
@@ -33,40 +33,41 @@ class LectureDocumentView(DocumentViewSet):
         FilteringFilterBackend,
         OrderingFilterBackend,
         DefaultOrderingFilterBackend,
-        SearchFilterBackend,
+        SimpleQueryStringSearchFilterBackend,
     ]
 
     simple_query_string_options = {
         "default_operator": "and",
     }
 
-    search_fields = {
-        "title": {
-            "boost": 5,
-        },
-        "abstract": {"boost": 1},
-        "type": {"boost": 5},
-        "keywords": {"boost": 6},
-        "sponsor": {"boost": 10},
-        "speaker": {"boost": 10},
-        "series": {"boost": 10},
-        "subject_category": {"boost": 10},
-    }
-
     filter_fields = {
         "lecture_id": None,
         "type": None,
         "keywords": None,
         "series": None,
         "sponsor": None,
-        "speaker": None
+        "speaker": None,
+        "subject_category": None,
     }
 
     ordering_fields = {
         "lecture_id": None,
         "title": None,
         "date": None,
         "score": None,
+        "series": None,
+        "subject_category": None
+    }
+
+    simple_query_string_search_fields = {
+        "title": {"boost": 5},
+        "abstract": {"boost": 1},
+        "type": {"boost": 5},
+        "keywords": {"boost": 6},
+        "sponsor": {"boost": 10, "proximity": 3},
+        "speaker": {"boost": 10},
```

Comment:
shouldn't be also `proximity: 3`?

Reply:
Should be nothing to be honest, "proximity" was a test of mine but it doesn't do anything, I'll remove

Reply:
👍 didn't check the docs